### ISSUE TITLE
Accessibility: Keyboard focus indicator is not clearly visible on list options.

### ISSUE THREAD

**Issue by formulahendry** [microsoft/vscode#212318]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.89.0
- OS Version: Linux- Ubuntu 22.04.4

Copied from: https://github.com/microsoft/vscode-azureapicenter/issues/109

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### GitHub Tags
#A11yTCS; #A11yMAS; #A11ySev2; #WCAG2.4.7; #GH_VSCodeextensionforAPICenter_Linux_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Keyboard; #Linux; #FTP; 

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Linux- Ubuntu 22.04.4

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. List options will appear.
7. Navigate using down/up arrow key and observe that Keyboard focus indicator is not clearly visible on list options.

### Actual Result:
Keyboard focus indicator is not clearly visible on list options when we navigate using up/down arrow key.

### Expected Result:
Keyboard focus indicator should be clearly visible on list options when we navigate using up/down arrow key.

### User Impact:
People with low vision often have difficulty perceiving tab focus that have insufficient contrast if Luminosity contrast ratio of visual focus indicator fails to meet the required ratio of 3:1. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a lightness difference of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors.

### Attachment:

![109_A11y_VS Code extension for API Center_APICenter_DefinitionFeature_Keyboard](https://github.com/microsoft/vscode-azureapicenter/assets/90901519/32697372-35e0-407b-99c9-1c5f2ce81f4d)

**Comment by daviddossett** [comment]
@sbatten how do we make changes to context menus in Linux? I thought we used custom menus for windows/web/linux, but it looks like that's not correct.

**Comment by formulahendry** [comment]
@daviddossett Do we have an ETA of this issue? Thx!
